src/components/layouts/Guided.js

```diff
@@ -10,6 +10,7 @@ import * as layoutTypes from '../../constants/LayoutTypes.js';
 export const Guided = ({ editor, user, scene, editorActions, authActions, projectActions, projects, courseActions, courses, course, match, sceneActions, classroomActions, classrooms }) => (
     <div className="App">
         <Header
+            viewOnly={scene.settings.viewOnly}
```

Comment:
Why does the header need this?

Reply:
For the handleSave function, I previously made it so it would check to see if its in view only so it knows to get the text from the editor or from the state. This fixed the bug where you couldn't save in view only